packages/contracts/graphql/get-non-flagship-projects-null-overrides.graphql

```diff
@@ -0,0 +1,15 @@
+query GetNonFlagshipProjectsNullOverrides {
+  projects_metadata(
+    where: {
+      script_type_and_version_override: { _is_null: true }
```

Comment:
@yoshiwarab curious if you think there's a better query to get the applicable projects 🤔